src/server.js

```diff
@@ -192,10 +201,10 @@ app.use("/v1/sites/:siteName", authenticatedSitesSubrouterV1)
 app.use("/v1", authenticatedSubrouterV1)
 
 app.use("/v2/auth", authV2Router.getRouter())
-// Endpoints which have siteName, used to inject site access token
-app.use("/v2/sites/:siteName", authenticatedSitesSubrouterV2)
 // Endpoints which have require login, but not site access token
 app.use("/v2", authenticatedSubrouterV2)
+// Endpoints which modify the github repo, used to inject site access token
+app.use("/v2/sites/:siteName", authenticatedSitesSubrouterV2)
```

Comment:
no strong opinion, just curious why do here rather than the init method in `identity/index`

Reply:
This was what was already done prior to this PR - I only rearranged the position so that it ends up after `/v2` endpoints! The reason for that was because of the new notifications middleware, which would cause issues for `/v2/site/:siteName` endpoints which belong in `authenticatedSubrouter` instead (such as collaborators and review requests)

I think we had this just because everything in that subrouter uses the same prefix!